src/main/java/net/flintmc/gradle/FlintGradlePlugin.java

```diff
@@ -308,4 +312,8 @@ public MavenArtifactURLCache getMavenArtifactURLCache() {
   public Project getProject() {
     return project;
   }
+
+  public MavenArtifactDownloader getDownloader() {
```

Comment:
Missing documentation (as far as I'm aware, this is null if operating in offline mode, indicate that in the documentation if thats the case!)